microsoft-edge/webview2/concepts/process-model.md

```diff
@@ -9,71 +9,96 @@ ms.prod: microsoft-edge
+The WebView2 Runtime uses the same process model as the Microsoft Edge browser.  This browser process model is described at [Inside look at modern web browser (part 1)][GoogleDeveloperWebUpdates201809InsideBrowserPart1BrowserArchitecture].
 
-One browser process is associated with the renderer processes and other utility processes as described in that article.  Additionally, if WebView2 is specified, the host app requesting processes use WebView2.  
+
+<!-- ====================================================================== -->
+## Processes in the WebView2 Runtime
+
+A collection of WebView2 Runtime processes, or WebView2 Process Group, is composed of the following processes:
+*  A single browser process.
+*  One or more render processes.
+*  Other helper processes, such as the GPU process and the Audio service process.
 
 :::image type="complex" source="../media/process-model-1.png" alt-text="Process 1" lightbox="../media/process-model-1.png":::
-   Process 1  
-:::image-end:::    
+   Process 1
+:::image-end:::
+
+Other than the browser process, the number and presence of processes in this collection can change as a WebView2 application makes use of WebView2 features.  For example, creating a new WebView from the same `CoreWebView2Environment` with a different domain in the `Source` property will usually start a new renderer process.  The logic that controls when these additional processes are created depends on the Chromium architecture, and is beyond the scope of the WebView2 Runtime.
+
+For example, the number of renderer processes varies based on the following conditions:
+*   Use of the _Site Isolation_ feature in the WebView2 Runtime.  See [Per-frame renderer processes - Site Isolation](https://developers.google.com/web/updates/2018/09/inside-browser-part1#site-isolation).
+*   The number of distinct disconnected origins that are rendered in instances of WebView2 that use the same user data folder.
+
+
+<!-- ====================================================================== -->
+## WebView2 Runtime processes and the user data folder
 
-A browser process is associated with only one user data folder.  A request process may specify more than one user data folder.  A request process that specifies more than one user data folder is associated with the same number of browser processes.  
-For example, a request process that requests access to two user data folders uses two browser processes.  
+All processes in a WebView2 Runtime processes collection are tied to the browser process, which in turn is associated with a single user data folder (UDF).  If an application makes use of multiple user data folders, a WebView2 Runtime processes collection will be created for each of these user data folders. 
```

Comment:
~~a WebView2 Runtime processes collection will be created~~ -> "a collection of WebView2 Runtime processes will be created" might be easier to read.